SCons/Environment.xml

```diff
@@ -2564,34 +2568,18 @@ env = Environment(platform=Platform('win32'))
 </example_commands>
 
 <para>
-The
-&f-env-Platform;
-form applies the callable object for the specified platform
-<parameter>string</parameter>
-to the environment through which the method was called.
+When called as a method of an environment,
```

Comment:
why swap out the &F-env-Platform?  Won't that resolve to `env.Platform()` ?  That'd be clearer.

Reply:
doesn't need to link to itself (we're already in the section that would point to)

Reply:
hmm can we specify as "called as env.Platform(plat)"  instead of "as a method of an environment"?
Sadly not all will know what method of environment means.

Reply:
checked - it's defined at the head of the section these appear in, here: https://scons.org/doc/production/HTML/scons-man.html#methods_and_functions_to_do_things